### DISCUSSION TITLE
How to prevent default skeleton.css from loading?

### DISCUSSION THREAD

**Discussion by devtrix** [skeletonlabs/skeleton#1886] [Q&A] [answered]
When I look in the browser dev console I see the following line in the <head> section
<style type="text/css" data-vite-dev-id="C:/Users/.../node_modules/@skeletonlabs/skeleton/dist/themes/theme-skeleton.css">:root...

I have tried everything to identity this.  I am still raw with Vite.  Please help.

**Comment by endigo9740** [comment, marked as answer]
Hey @devtrix, in Skeleton v1.x that theme stylesheet is typically setup in your root layout in `/src/routes/+layout.svelte`. If you're wanting run a Skeleton v1 project, you can remove that theme, but make sure to replace it with another theme if you want Skeleton to operate.

Themes are controlled in a different manner in Skeleton v2 though, so if you need help with that let us know.

**Reply by devtrix** [reply]
You guys are the best.  I didnt expect a quick answer no matter what.  Thank you for that.  Also I feel so dumb I seee that now.  I come from server side .net and too many things to learn, regardless it all makes sense now.
Thanks a million once again.  That fixed it.

**Reply by devtrix** [reply]
Also  wanted to ask, can I use the v2 RC in production.?

**Reply by endigo9740** [reply]
The release notes for v2 are here, if you would like to see what's new:
https://github.com/skeletonlabs/skeleton/discussions/1845

And the migration guide is here:
https://github.com/skeletonlabs/skeleton/discussions/1854

Just keep in mind it's a "release candidate", which means we're testing it before we consider it stable. There may be some minor changes. So if you're running in production you -may- want to wait. But we anticipate the v2 stable release in about a week and a half.
